plugins/auto-sizes/readme.txt

```diff
@@ -48,6 +48,10 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 == Changelog ==
 
+= 1.0.1 =
+
+* Update tested WordPress version to 6.5. ([1027](https://github.com/WordPress/performance/pull/1027))
```

Comment:
It's too bad a version bump is needed to do just this. Previously I've just manually bumped the version in the `readme.txt` of the latest SVN tag as a way to avoid having to do a release.

Reply:
Agreed. I think there's a way to automate this too via GitHub workflow somehow. But until we have this, probably fine to do this.